saleor/graphql/attribute/constants.py

```diff
@@ -0,0 +1,26 @@
+from ...permission.enums import (
+    PagePermissions,
+    ProductPermissions,
+    ProductTypePermissions,
+)
+from .enums import AttributeTypeEnum
+
+PERMISSIONS_MAP: dict = {
+    AttributeTypeEnum.PRODUCT_TYPE.value: {
+        "default": (
+            ProductPermissions.MANAGE_PRODUCTS,
+            ProductTypePermissions.MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES,
+        )
+    },
+    AttributeTypeEnum.PAGE_TYPE.value: {
+        "default": (
+            PagePermissions.MANAGE_PAGES,
+            ProductTypePermissions.MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES,
+        ),
+        "create": (
+            ProductPermissions.MANAGE_PRODUCTS,
+            PagePermissions.MANAGE_PAGES,
+            ProductTypePermissions.MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES,
+        ),
+    },
+}
```

Comment:
Shouldn't this be:
```

PERMISSIONS_MAP: dict = {
    AttributeTypeEnum.PRODUCT_TYPE.value: {
        "default": (
            ProductPermissions.MANAGE_PRODUCTS,
            ProductTypePermissions.MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES,
        )
    },
    AttributeTypeEnum.PAGE_TYPE.value: {
        "default": (
            PagePermissions.MANAGE_PAGES,
            ProductTypePermissions.MANAGE_PAGE_TYPES_AND_ATTRIBUTES,
        ),
        "create": (
            ProductPermissions.MANAGE_PAGES,
            PagePermissions.MANAGE_PAGES,
            ProductTypePermissions.MANAGE_PAGE_TYPES_AND_ATTRIBUTES,
        ),
    },
}
```

We do have `MANAGE_PAGE_TYPES_AND_ATTRIBUTES` and so we should utilise it here 🙌